openquake/hazardlib/geo/geodetic.py

```diff
@@ -416,7 +416,7 @@ def npoints_towards(lon, lat, depth, azimuth, hdist, vdist, npoints):
     hdists = numpy.arange(npoints, dtype=float)
     hdists *= (hdist / EARTH_RADIUS) / (npoints - 1)
     vdists = numpy.arange(npoints, dtype=float)
-    vdists *= vdist / (npoints - 1)
+    vdists *= float(vdist) / (npoints - 1)
 
```

Comment:
The error here is not the casting. The error is in the division operator. It has been fixed in Python 3, so the right solution is to add a line at the beginning of the module

```python
from __future__ import division
```